move migration docs (#192)
cossacklabs/product-docs: acra/configuring-maintaining/migrations/migration-0.77.0-0.80.0.md

---
weight: 1

> title:  Updating Acra from 0.77.0 to 0.80.0

Reviewer: ```suggestion
title: Updating Acra from 0.77.0 to 0.80.0
```

---

# Migration guide

Please read the migration guide if at some point you used Acra version 0.77.0 - in Acra 0.80.0 some radical renaming was applied throughout. 

> Here is the detailed migration guide.

Reviewer: ```suggestion
```

If you were using Acra before `0.80.0` release you might want to update configuration files and parameter names.

## The main services:

| Old name | New name | Function |
| --- | --- | --- |
| AcraServer | AcraServer | decrypts data from database |
| AcraWriter | AcraWriter | encrypts data on client side |
| AcraProxy | AcraConnector | encrypts traffic between client and server using Themis Secure Session |
| AcraCensor | AcraCensor | firewall, part of AcraServer, blocks suspicious SQL requests to the database |
| AcraConfigUI | AcraWebConfig | lightweight HTTP web server for managing AcraServer's certain configuration options |

## Utilities:

| Old name | New name | Function |
| --- | --- | --- |
| acra_rollback | AcraRollback | decrypts the whole database |
| acra_genkeys | AcraKeymaker | generates encryption keys for storage and transport for Acra components |
| acra_genauth | AcraAuthmanager | generates user accounts for AcraWebConfig |
| acra_genpoisonrecord | AcraPoisonRecordMaker | generates poision records for databases |
| acra_addzone | AcraAddzone | generates Zones header for AcraWriter |

## Configuration parameters

### AcraServer configuration

> Check out the full guide to [AcraServer](/acra/what-is-acra/#how-acraserver-works).

Reviewer: ```suggestion
Check out the full guide to [AcraServer](/acra/acra-in-depth/architecture/acraserver/).
```
